### PR TITLE
#129: skip logging NotFound error of game state dump at startup

### PR DESCRIPTION
**Pull request by cai-jack**

### CODE REVIEW

File: backend/src/main.rs [hunk 1]
```diff
@@ -173,7 +174,11 @@ async fn main() {
             }
         }
         Err(e) => {
-            error!(init_logger, "Failed to open dump"; "error" => format!("{:?}", e));
+            if let ErrorKind::NotFound = e.kind() {
+                // println!("file not found, not logging");
```
Comment (rbtying) [inline backend/src/main.rs:178]: drop the `println!`?

otherwise, sgtm

Comment (cai-jack) [reply backend/src/main.rs]: K. fixed. I could not find a "if not let" to do this. Do u?

Comment (rbtying) [reply backend/src/main.rs]: You can use a match, if you'd like:

```
match e.kind() {
  ErrorKind::NotFound => (),
  _ => {
   // log
  }
}
```

Comment (rbtying) [reply backend/src/main.rs]: Actually, `ErrorKind` implements `Eq`. So you can just do

```
if e.kind() != ErrorKind::NotFound {

}
```

Comment (cai-jack) [reply backend/src/main.rs]: Cool. will do. Thanks

**Pull request merged**